node/send_receive/lib/eventdata.js

```diff
@@ -6,9 +6,10 @@
 /**
  * @class EventData
  * @classdesc Constructs a {@linkcode EventData} object.
- * @param {String}  body   The event payload as a byte array.
+ * @param {object} link The receiver link for providing settlment methods.
+ * @param {object}  message The message object containing body and messageAnnotations.
  */
-function EventData(body, systemProperties) {
+function EventData(link, message) {
```

Comment:
For IoT Hub right now we use the client. the receiver object is "hidden" and it's unique anyway. This being said when the receiver was exposed as a public API we had moved these to the receiver. I don't believe in the case of event hubs that the client knows which link to use to settle the message but the receiver does... so I'm thinking receiver? That way we don't have to carry around in the message the reference to the link?